orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/ClientSession.java

```diff
@@ -55,6 +59,7 @@ public VectorOfTrust getEffectiveVectorOfTrust() {
 
```

Comment:
Maybe add a getter method for the vtrList here? It will be needed in another ticket and I think it makes sense to have it as part of the vtrList field set-up